README.md

```diff
@@ -1,135 +1,69 @@
- - IDEAS.Buildings.Examples (Simple examples)
- - IDEAS.Examples.TwinHouses (The Holzkirchen twin house validation experiment)
- - IDEAS.Examples.PPD12 (A model of a terraced house, including heating and ventilation.)
- - IDEAS.Examples.IBPSA (Models for BOPTEST.)
- - IDEAS.Examples.Tutorial (A tutorial.)  
-See the documentation sections of the respective models for more details.
+The IDEAS library extends from the [Modelica IBPSA library](https://github.com/ibpsa/modelica-ibpsa) and has a particular focus
+on the development of detailed building models. It is one of the four sister libraries that extend from the core Modelica IBPSA
+library, alongside [AixLib](https://github.com/RWTH-EBC/AixLib), [Modelica Buildings](https://github.com/lbl-srg/modelica-buildings),
+and [BuildingSystems](https://github.com/UdK-VPT/BuildingSystems).
+
+## License
+IDEAS is licensed by [KU Leuven](http://www.kuleuven.be) and [3E](http://www.3e.eu) under a 
+[BSD 3 license](https://htmlpreview.github.io/?https://github.com/open-ideas/IDEAS/blob/master/IDEAS/legal.html).
+
+## Community and contributions
+We love to hear what you are using IDEAS for. You are welcome to open an issue on GitHub or contact the development team via
+email to share your feedback. If you like our library, you can support IDEAS by starring it at the top right of our Github page.
+Bug reports and feature suggestions can be submitted as [GitHub issues](https://github.com/open-ideas/IDEAS/issues), and
+contributions in the form of [pull requests](https://github.com/open-ideas/IDEAS/pulls) are highly encouraged. Before submitting
+a pull request, please consult the the IDEAS
+[Style Guide and Conventions](https://github.com/open-ideas/IDEAS/wiki/Style%20Guide%20and%20GitHub%20Good%20Practice)
+to ensure consistency with the project's conventions.
+
+The IDEAS library was originally developed by KU Leuven and [3E](https://3e.eu), and is currently developed and maintained by
+the [Thermal Systems Simulation (The SySi)](http://the.sysi.be) research group of KU Leuven. 
+The library includes significant contributions by the [Building Physics and Sustainable Design Section](https://bwk.kuleuven.be/bwf)
+of KU Leuven, the [Building Physics Research Group](https://www.ugent.be/ea/architectuur/en/research/research-groups/building-physics)
+of UGent, [IBPSA project 1](https://ibpsa.github.io/project1/), [IEA EBC Annex 60](https://iea-annex60.org) and the
+[Electrical Energy Systems and Applications Section](https://www.esat.kuleuven.be/electa) of KU Leuven.
 
 ## Tool support
-Our goal is to provide support for Dymola and OpenModelica. Consequently, any tool that supports the full Modelica specification should be able to run our models. Feel free to file a bug report in case we do not adhere to the Modelica specification.
+IDEAS is fully compliant with the [Modelica Specification](https://specification.modelica.org/), and is therefore, in
```

Comment:
"fully complaint" is a dangerous statement. We test our developments in Dymola, which sometimes allows deviations from the Modelica Standard (see e.g. https://github.com/open-ideas/IDEAS/issues/1317, which is an incompatibility that we solved in the past). "Largely compliant" seems more correct to me.